docs/index.bs

```diff
@@ -1986,78 +1983,64 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       <dfn method for="Cache"><code>add(|request|)</code></dfn> method *must* run these steps:
 
         1. Let |requests| be an array containing only |request|.
-        1. Set |responseArrayPromise| to the result of running the algorithm specified in {{Cache/addAll(requests)}} passing |requests| as the argument.
-        1. Return the result of <a>transforming</a> |responseArrayPromise| with a fulfillment handler that returns undefined.
+        1. Let |responseArrayPromise| be the result of running the algorithm specified in {{Cache/addAll(requests)}} passing |requests| as the argument.
+        1. Return the result of [=transforming=] |responseArrayPromise| with a fulfillment handler that returns undefined.
     </section>
 
     <section algorithm="cache-addAll">
       <h4 id="cache-addAll">{{Cache/addAll(requests)}}</h4>
 
       <dfn method for="Cache"><code>addAll(|requests|)</code></dfn> method *must* run these steps:
 
-        1. Let |responsePromiseArray| be an empty array.
-        1. Let |requestArray| be an empty array.
+        1. Let |r| be null.
+        1. Let |responsePromises| be an empty [=list=].
+        1. Let |requestList| be an empty [=list=].
         1. For each |request| whose type is {{Request}} in |requests|:
```

Comment:
Is this so we reject without consuming the body of the request? Might be worth adding a note about that.

Reply:
I don't think we changed it so for that particular reason. The only apparent reason I recall is to make it return early when requests with a non-GET method come in. I'll inspect how we can improve/simplify this part of the algorithm as a separate work.